pkg/kubectl/cmd/apply_test.go

```diff
@@ -136,9 +147,16 @@ func annotateRuntimeObject(t *testing.T, originalObj, currentObj runtime.Object,
 	if currentAnnotations == nil {
 		currentAnnotations = make(map[string]string)
 	}
-	currentAnnotations[annotations.LastAppliedConfigAnnotation] = string(original)
+
+	originalAccessor.SetAnnotations(map[string]string{})
+	originalWithEmptyAnno, err := runtime.Encode(codec, originalObj)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	currentAnnotations[annotations.LastAppliedConfigAnnotation] = string(originalWithEmptyAnno)
```

Comment:
Do you want me to add a comment ? 
I do not know the actual reason in the implementation. 
I have looked into the logs of kubectl and found this difference. Adding the same field here made the PATCH requests the same. 
Does writing just that in a comment add enough information to the code? 

Do you have an intelligent blob for me to add as a comment ?